packages/blocks/src/api/test/registration.js

```diff
@@ -1607,7 +1607,7 @@ describe( 'blocks', () => {
 			expect( source.setValue ).toBeUndefined();
 			expect( source.setValues ).toBeUndefined();
 			expect( source.getPlaceholder ).toBeUndefined();
-			expect( source.canUserEditValue() ).toBe( false );
+			expect( source.canUserEditValue ).toBeUndefined();
```

Comment:
There's also `.toBeFalsy()` (which is a slightly looser assertion and would work both with and without the other changes in this PR, thus "bridging" the change).

It's also a really small detail, so feel free to ignore 🤷‍♂️ 

```suggestion
			expect( source.canUserEditValue ).toBeFalsy();
```

Reply:
I didn't know about that, thanks for the tip 🙂 On the other hand, I prefer keeping it `undefined` to keep consistency with the rest of the properties, and because it is actually expected to be `undefined` and not false after these changes.